src/explorer/AppServiceNode.ts

```diff
@@ -40,15 +40,19 @@ export class AppServiceNode extends SiteNodeBase {
         }
 
         const treeDataProvider = this.getTreeDataProvider<AppServiceDataProvider>();
-
+        const nodes = [];
+        const appServicePlan = await this.getAppServicePlan();
+        if (appServicePlan.sku.tier !== 'Basic') {
+            // Basic Service Plans do not support deployment slots
```

Comment:
I think we usually 'gray things out' if something isn't supported, rather than removing it completely. Since we can't 'gray it out', maybe we just show a message that says 'unsupported on basic plan' in the tree? Something analagous to the portal:

![screen shot 2017-10-30 at 3 03 56 pm](https://user-images.githubusercontent.com/11282622/32197994-9503b762-bd83-11e7-8b5a-2c7b7cbea9ee.png)

Reply:
FYI - I'm still waiting on this before I'll approve the PR. If you feel strongly, I will also approve if you file an issue for the 0.4.0 milestone and we can discuss our options there

Reply:
Here's my current working solution.  I welcome suggestions for the Node Label.

![image](https://user-images.githubusercontent.com/5290572/32356807-e88cca04-bff4-11e7-8334-946b5cef920c.png)

![image](https://user-images.githubusercontent.com/5290572/32356819-f9476a84-bff4-11e7-8737-f197796e412a.png)

Reply:
Child node language copies the portal's (pictured above)

Reply:
I might put 'N/A for Basic Service Plan' in parentheses, but otherwise looks good to me